activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ModelServiceImpl.java

```diff
@@ -199,7 +199,15 @@ public Model updateModel(Model modelToBeUpdated,
 
     @Override
     public Model copyModel(Model modelToBeCopied, Project project) {
-        return modelRepository.copyModel(modelToBeCopied, project);
+        Model copiedModel = modelRepository.copyModel(modelToBeCopied, project);
+        modelIdentifiers.put(String.join("-",
```

Comment:
Consider extracting a constant for the separator character.

Reply:
Moved to constant